### DISCUSSION TITLE
g++ compile on macOS

### DISCUSSION THREAD

**Discussion by BartJagodzinski** [bitcoin-core/secp256k1#1208] [Q&A] [answered]
Hello, on Linux to compile a .cpp file with g++ I just use -lsecpk256k1 flag. On macOS Im getting error "library not found for -lsecp256k1". What am I doing wrong?

**Comment by real-or-random** [comment, marked as answer]
Have you installed libsecp256k1 on your macOS machine?

**Reply by BartJagodzinski** [reply]
Yes, restart of macbook solved the issue. Thank you for fast reply.

**Comment by neeraj542** [comment]
Hello everyone,

I hope this message finds you well. My name is Neeraj and I am a contributor to your repository. I am reaching out to express my interest in contributing to the Ledger Organization in the Summer of Bitcoin program 2023.

As someone who is passionate about blockchain technology and its potential to revolutionize finance, I believe that this program presents a great opportunity for me to develop my skills and make a meaningful contribution to the community.

I am excited to start contributing, but I have some doubts about where to begin. Could you please guide me on where I can ask my doubts and how I can start contributing to the Ledger Organization's project?

I appreciate your time and thank you in advance for your help. I am looking forward to working with you and the rest of the community.

Best regards,
Neeraj Meena

**Reply by real-or-random** [reply]
Hi Neeraj. Sorry, we cannot help you. We're the libsecp256k1 library, which is not affiliated with Ledger. You can find the Ledger organization here: https://github.com/LedgerHQ.

**Comment by neeraj542** [comment]
Dear Mentor, I wanted to let you know that I am also participating in this library (libsecp256k1) i.e, adaptor signature and I'm excited to be a part of it. I'm looking forward to learning new things, sharing my knowledge, and connecting with other participants. Thank you for being a part of this program and for supporting all of us as we work towards our goals. Do you have any advice or recommendations for getting the most out of this library? I'm eager to make the most of this opportunity.
